### ISSUE TITLE
disable or change Style/Next

### ISSUE THREAD

**Issue by searls** [testdouble/standard#56]
It's really irritating when in a block and:

``` ruby
do |lol|
  if lol.good?
    # do stuff
  end
end
```

is corrected as wrong with:

``` ruby
do |lol|
  next unless lol.good?
  # do stuff
end
```

If this rule only applied to first line guard clauses I'd be cool, but sometimes you're 5 lines into a block and suddenly it's being broken out of with a "next" halfway down

**Comment by twe4ked** [comment]
I'd like to see this disabled either way. I'm not a huge fan of enforcing guard clauses for anything. Sometimes it makes more sense to use an if statement for readability. I don't have a good example off the top of my head, sorry.

[Docs for reference](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/Next).

**Comment by searls** [comment]
Yeah I am inclined to agree. Will mark this approved, to be changed next time I sit down to work on Standard

**Comment by searls** [comment]
Landed in 0.0.22
